js/app.js

```diff
@@ -0,0 +1,182 @@
+// 1
+function setAlarm(employed, vacation){
+    return employed && vacation === false;
+}
+
+// 2
+function closeCompare(a, b, margin=0) {
+    if (a >= b - margin && a <= b + margin) {
```

Comment:
It is more readable when you compare the equation to a number
Good:
`age - 18 < 0`
Bad:
`0 > age - 18`